rcsb/utils/insilico3d/ModelArchiveModelProvider.py

```diff
@@ -92,17 +94,29 @@ def __reload(self, **kwargs):
             startTime = time.time()
             startDateTime = datetime.datetime.now().isoformat()
             useCache = kwargs.get("useCache", True)
-            baseUrl = kwargs.get("baseUrl", "https://www.modelarchive.org/api/projects")
+            baseUrl = kwargs.get("baseUrl", self.__modelArchiveSummaryPageBaseApiUrl)
             modelArchiveRequestedDatasetD = kwargs.get("modelArchiveRequestedDatasetD", {})
-            if not modelArchiveRequestedDatasetD:  # Fill in default
+            if not modelArchiveRequestedDatasetD:  # use default
                 modelArchiveRequestedDatasetD = {
                     "ma-bak-cepc": {
                         # "bulkFileName": "ma-bak-cepc.zip",
-                        "numModelsTotal": 1106,  # Necessary for forming filenames
+                        # "numModelsTotal": 1106,
                     },
                     "ma-ornl-sphdiv": {
                         # "bulkFileName": "ma-ornl-sphdiv.zip",
-                        "numModelsTotal": 25134,
+                        # "numModelsTotal": 25134,
+                    },
+                    "ma-coffe-slac": {
+                        # "bulkFileName": "",
+                        # "numModelsTotal": ,
+                    },
+                    "ma-asfv-asfvg": {
+                        # "bulkFileName": "ma-asfv-asfvg.zip",
+                        # "numModelsTotal": ,
+                    },
+                    "ma-t3vr3": {
+                        # "bulkFileName": "ma-t3vr3.zip",
+                        # "numModelsTotal": ,
```

Comment:
"numModelsTotal" seems to have been replaced everywhere else. Is it intentional to leave it here commented?

Reply:
Good point, I'll clean that up too. Thanks.